src/components/views/RealTimeChart.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Plot from 'react-plotly.js';
+
+import './css/RealTimeChart.css';
+
+
+export default props => {
+    return (
+        <Plot
+            data={props.data}
+            layout={{
+                width: 600,
+                height: 250,
+                yaxis: {showline: true},
+                title: props.title
+            }}
+        />
+    );
+};
```

Comment:
Small stylistic thing. Since the function doesn't do anything but return, we could rewrite this as:

```javascript
export default props => (
    <Plot
        data={props.data}
        layout={{
            width: 600,
            height: 250,
            yaxis: {showline: true},
            title: props.title
        }}
    />
);
```